openwisp_monitoring/device/utils.py

```diff
@@ -1,9 +1,24 @@
+from django.conf import settings
+from swapper import load_model
+
 from ..db import timeseries_db
 from . import settings as app_settings
+from .signals import device_status_unknown
 
 SHORT_RP = 'short'
 DEFAULT_RP = 'autogen'
 
+Device = load_model('config', 'Device')
+DeviceMonitoring = load_model('device_monitoring', 'DeviceMonitoring')
+
+
+def handle_critical_check_change(check):
```

Comment:
So all this code logic should go in a class method on the `DeviceData` model as suggested earlier.